### PR TITLE
Added a PCP article.

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Moving PCP from Micro admin to a separate article.

### CODE REVIEW

**Comment by lvicoun** [comment]
[pcp-container_en.pdf](https://github.com/SUSE/doc-modular/files/14790567/pcp-container_en.pdf)

File: articles/performance-copilot-metrics.asm.xml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+              You want to monitor performance on your system and this article provides infomration
+              how to configure and use the toolkit.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>EFFORT</term>
+            <listitem>
+              <para>
+                What's the effort one has to put in?
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml]: I know that you know it's missing :-)

File: articles/performance-copilot-metrics.asm.xml [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+      <description>Legal Notice</description>
+    </resource>
+    <resource href="../common/license_gfdl1.2.xml" xml:id="_gfdl">
+      <description>GNU Free Documentation License</description>
+    </resource>
+  </resources>
+  <!-- S T R U C T U R E -->
+  <structure renderas="article" xml:id="pcp-container" xml:lang="en">
+    <merge>
+      <title>Analysing performance metrics using the Performance Co-Pilot Analysis Toolkit</title>
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml:74]:
```suggestion
      <title>Analyzing Performance Metrics Using the Performance Co-Pilot Analysis Toolkit</title>
```

File: articles/performance-copilot-metrics.asm.xml [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        in a container. 
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>WHY?</term>
+          <listitem>
+            <para>
+              You want to monitor performance on your system and this article provides infomration
+              how to configure and use the toolkit.
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml:162]:
```suggestion
              You want to monitor performance on your system, and this article provides information
              on how to configure and use the toolkit.
```

File: articles/performance-copilot-metrics.asm.xml [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>EFFORT</term>
+            <listitem>
+              <para>
+                It takes approximatelly 40 minutes of reading time.
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml:170]:
```suggestion
                It takes approximately 40 minutes of reading time.
```

File: articles/performance-copilot-metrics.asm.xml [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+              </para>
+            </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>GOAL</term>
+            <listitem>
+              <para>
+                You will be able to start the PCP container with configuration that suits your needs.
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml:178]:
```suggestion
                You will be able to start the PCP container with a configuration that suits your needs.
```

File: articles/performance-copilot-metrics.asm.xml [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+      <meta name="architecture">
+        <phrase>&x86-64;</phrase>        
+      </meta>
+      <!-- enter one or more product names and version -->
+      <meta name="productname" its:translate="no">
+        <productname version="6.0">&productname;</productname>
+      </meta>
+      <meta name="title" its:translate="yes">Performance Co-Pilot analysis toolkit</meta>
+      <meta name="description" its:translate="yes">Toolkit that collects performance metrics from
+      the system.</meta>
```
Comment (dariavladykina) [inline articles/performance-copilot-metrics.asm.xml:122]:
```suggestion
      <meta name="description" its:translate="yes">The toolkit that collects performance metrics from
      the system</meta>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Pull request merged**